File: pset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


height = 0
height = get_int("Height: ")
while(height < 1 or height > 8):
    height = get_int("Height: ")
for i in range(height):
    j = height
    while(j >= 1):
        if(j > i + 1):
            print(" ", end="")
        else:
            print("#", end="")

        j = j - 1
    print("  ", end="")
    k = 1
    while(k <= i + 1):
        if(k <= i + 1):
            print("#", end="")
        else:
            print(" ", end="")

        k = k + 1
    print()

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = 0;
    while (n < 1 || n > 8)
    {
        n = get_int("Height: ");
    }

    for (int i = 1; i <= n; i++)
    {
        for (int j = n; j >= 1; j--)
        {
            if (j > i)
            {
                printf(" ");
            }

            else
            {
                printf("#");
            }
        }

        printf("  ");

        for (int k = 1; k <= i; k++)
        {

            if (k <= i)
            {
                printf("#");
            }

            else
            {
                printf(" ");
            }

        }

        printf("\n");




    }

}